src/test/java/com/puppycrawl/tools/checkstyle/bdd/InlineConfigParser.java

```diff
@@ -115,11 +117,21 @@ private static void setCheckProperties(InputConfiguration.Builder inputConfigBui
         }
     }
 
-    private static void setViolationLineNumbers(InputConfiguration.Builder inputConfigBuilder,
-                                    List<String> lines) {
+    private static void setViolationLines(InputConfiguration.Builder inputConfigBuilder,
+                                          List<String> lines) {
         for (int lineNo = 2; lineNo < lines.size(); lineNo++) {
             if (VIOLATION_PATTERN.matcher(lines.get(lineNo)).matches()) {
-                inputConfigBuilder.addViolation(lineNo + 1);
+                inputConfigBuilder.addViolationLine(lineNo + 1);
+            }
+        }
+    }
+
+    private static void setViolationMessages(InputConfiguration.Builder inputConfigBuilder,
```

Comment:
it is strange, that we store line numbers and messages separately, maybe we should combine them in on object?

Reply:
what if we have multiple violations on one line?

Reply:
it is ok, I think we should not support violation messages for multiple violations or rethink comment patten

but it will be confusing, if list of violation lines and list of violation messages will have different sizes and they will have different comparison logic with actual violations

Reply:
done